app/controllers/devise_token_auth/registrations_controller.rb

```diff
@@ -3,19 +3,11 @@ class RegistrationsController < DeviseTokenAuth::ApplicationController
     before_action :set_user_by_token, only: [:destroy, :update]
     before_action :validate_sign_up_params, only: :create
     before_action :validate_account_update_params, only: :update
+    before_action :build_resource_for_registration, only: [:create]
```

Comment:
I believe that would be clearer if this is part of the method instead of a before filter, so right after `def create`, call `build_resource_for_registration`